chain/cosmos/chain_node.go

```diff
@@ -785,7 +784,7 @@ func (nodes ChainNodes) PeerString() string {
 		}
 		hostName := n.HostName()
 		ps := fmt.Sprintf("%s@%s:26656", id, hostName)
-		nodes.logger().WithField("container", n.Name()).Infof("(%s) peering (%s)", hostName, ps)
+		nodes.logger().With("container", n.Name()).Infof("(%s) peering (%s)", hostName, ps)
```

Comment:
Move hostname and peer string(?) to fields.

Reply:
I agree. Like other parts of the codebase, this was the previous pattern.